src/app/components/charts/BarChart.tsx

```diff
@@ -21,10 +21,24 @@ interface BarChartProps<T extends object> extends Formatters {
   rounded?: boolean
   withBarBackground?: boolean
   withLabels?: boolean
-  tickMark?: boolean
   margin?: Margin
 }
 
+const calculateCustomTicks = <T extends object>(data: T[], dataKey: keyof T) => {
```

Comment:
How come we need custom tick positions?

Reply:
Still experimenting here and my plan is now to try to avoid custom ticks somehow.

Why I was testing custom ticks: with just a "domain" prop, `YAxis` can be rendered with gaps.  Might be related to `https://github.com/recharts/recharts/issues/2019`. 
<img width="138" alt="Screenshot 2023-06-26 at 13 16 50" src="https://github.com/oasisprotocol/explorer/assets/891392/892f2d70-ad87-43f0-ad8c-646f9b3d87e6">

Reply:
:/

Reply:
Reproduced in https://github.com/oasisprotocol/explorer/pull/591 even with `interval="preserveStartEnd"`